app/controllers/alaveteli_pro/pricing_controller.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+# Does not inherit from AlaveteliPro::BaseController as is pre-login
+class AlaveteliPro::PricingController < ApplicationController
```

Comment:
Hmm, maybe this should be called `AlaveteliPro::PlansController` after looking at https://github.com/mysociety/alaveteli-professional/issues/331#issuecomment-324043946. I was going to add `AlaveteliPro::PlansController#show` for the "contract" page (`GET /plans/pro`), so it would make sense for `AlaveteliPro::PlansController#index` to be the view of all plans. We could add a custom route if we prefer `/pro/pricing` instead of `/pro/plans`.

Reply:
Superseded by https://github.com/mysociety/alaveteli/pull/4203